File: PYTHON/ecommercecheckoutsystem.py
print("""
		Semicolon Ecommerce Checkout System
""")

value_added_tax = 7.5 / 100

customer_name = input("What is the customer's name: ")

user_items = input("what did the user buy: ")

pieces = int(input("How many pieces: "))

per_units = float(input("How much per unit: "))



total = pieces * per_units

value_added_tax = 0.075 * total

total_with_vat = total + value_added_tax


print(f"The total is: {total}")

print(f"The value added tax is: {value_added_tax}")

File: JAVASCRIPT/EcommerceCheckoutSystem.js
const prompt = require('prompt-sync')();

console.log()

console.log("		Semicolon Ecommerce Checkout System")


let isRunning = true;

let customerName = prompt("What is the customer's name: ");

//while (isRunning) {

	let userItems = prompt("What did the user buy: ");

	let pieces = parseInt(prompt("How many pieces: "));

	let perUnits = parseFloat(prompt("How much per unit: "));


	let total = pieces * perUnits;

	let valueAddedTax = total * 0.075;

	let totalWithVat = total + valueAddedTax;



	console.log("Customer Name: " + customerName);	

	console.log("Item: $" + userItems);

	console.log("Quantity: $" + pieces);

	console.log("Price: $" + perUnits);

	console.log("Total: $" + total);

	console.log("The Value-added Tax (7 .5%) is: $" + valueAddedTax);

	console.log("Total with VAT: $" + totalWithVat);

	let addItems = prompt("Add more items? (1 for yes and 0 for no): ");


 /*   if (addItems == 0){
        isRunning = false;
    }
} */

	console.log("THANK YOU FOR YOUR PATRONAGE");

